### PR TITLE
Fixes #2166 revision of Hofbauer cell

### LINKED ISSUE #2166: [Class hierarchy]Hofbauer cell CL:0000235 is a tissue resident macrophage
**CL term**
Hofbauer cell CL:0000235

**Suggested revision of class hierarchy**
It should appear under tissue resident macrophage.

Thomas JR, Naidu P, Appios A, McGovern N. The Ontogeny and Function of Placental Macrophages. Front Immunol. 2021 Oct 21;12:771054. doi: 10.3389/fimmu.2021.771054. PMID: 34745147; PMCID: PMC8566952.
Quote from paper: "Hofbauer cells (HBC) are a population of tissue-resident macrophages found within human placental villi. These cells appear very early during development and have been identified at day [1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8566952/#B1)8 post-conception (1, [2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8566952/#B2)). HBC are the only significant immune cell population found within the normal healthy human placenta."

Screenshot 2023-10-02 at 4 03 33 PM

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #2166 revision of Hofbauer cell

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -28903,7 +28903,8 @@ AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_3000001 <https
 AnnotationAssertion(oboInOwl:creation_date obo:CL_3000001 "2019-02-18T19:41:20Z")
 AnnotationAssertion(oboInOwl:id obo:CL_3000001 "CL:3000001")
 AnnotationAssertion(rdfs:label obo:CL_3000001 "Hofbauer cell")
-SubClassOf(obo:CL_3000001 obo:CL_0000235)
+SubClassOf(obo:CL_3000001 obo:CL_0000864)
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:28906]: Consider adding dbxref annotation (provided in issue by requester).

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -28903,7 +28903,8 @@ AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_3000001 <https
...
+SubClassOf(obo:CL_3000001 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003124))
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:28907]: Text definition invokes "placenta" and "chorionic villus". Consider aligning.

**Review by bvarner-ebi — COMMENTED** [review]
Please see inline comments.
Will defer to assignee's judgment on whether to incorporate suggestions.

**Comment by aleixpuigb** [comment]
Thank you for your comments, @bvarner-ebi. 
This cell was automatically generated, so the definition is outdated, the reference is not adequate (the definition is from wikipedia, but it is not referenced) and human specific. It is also missing an abbreviated synonym (HBC). I will go back to the original ticket to propose further edits.

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:28907]: - "an heterogenic" --> "a heterogenic". ("a" precedes | hɛ |)
- "fetal derived" --> "fetal-derived" (compound adjective)
- "chorionic lavae" --> "chorion lavae" (cursory online check shows that the latter is more common)

**Review by bvarner-ebi — COMMENTED** [review]
Defer content review to SME.
Please see inline comments.

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:28907]: - "lavae --> "laeve" (tracing back to the [reference cited](https://pubmed.ncbi.nlm.nih.gov/30498493/) in the paper, it is "chorion laeve")

Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:28907]: All changes have been addressed

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**